documentation/src/main/asciidoc/topics/proc_using_jboss_marshaller.adoc

```diff
@@ -0,0 +1,41 @@
+[id='jboss_marshalling']
+= Using JBoss Marshalling
+JBoss Marshalling is a serialization-based marshalling library and was the
+default marshaller in previous {brandname} versions.
+
+[NOTE]
+====
+* You should use serialization-based marshalling with {brandname} but use
+Protostream instead, which is a high-performance binary wire format that
+ensures backwards compatibility.
+
+* JBoss Marshalling is deprecated in {brandname} and will be removed in a future
+release. Along with JBoss Marshalling {brandname} deprecates the
```

Comment:
" Along with JBoss Marshalling {brandname}" redundant? Can we just state that "JBoss Marshalling and the `AdvancedExternalizer` interface is deprecated in ..."